### DISCUSSION TITLE
Liquidsoap upgrade via Opam

### DISCUSSION THREAD

**Discussion by sorintodys** [savonet/liquidsoap#2512] [Installation]
Hello,

I try to upgrade liquidsoap with opam but is stuck to version 2.0.3-1 The other servers upgrade ok to 2.0.6
What to try for solve this issue?
![liquid](https://user-images.githubusercontent.com/12820659/178812978-65fabfda-4c09-4c26-9c6c-3f9742224cd4.png)

**Comment by caf2022** [comment]
Hi, how are you? You need to download the liquidsoap-2.0.6 and pin the directory for opam:

```
cd /root
wget https://github.com/savonet/liquidsoap/releases/download/v2.0.6/liquidsoap-2.0.6.tar.bz2
tar -xvf liquidsoap-2.0.6.tar.bz2
cd liquidsoap-2.0.6/
opam pin add liquidsoap . --no-action -y
opam upgrade
```

**Reply by sorintitu** [reply]
Hello!
Thank you very much for help, that was working ok.
Issue was ocaml old version. I remove ocaml and put opam to install liquidsoap again and is install liquidsoap 2.0.6
Best Regards,
Sorin
